docs/source/dataframe-create.rst

```diff
@@ -50,8 +50,26 @@ Pandas:
 .. autosummary::
     from_pandas
 
-Locations
----------
+Writing to remote locations
```

Comment:
I think reading should come before writing. It's probably more common.

Reply:
Reading through the document, I'd prefer to have all the "creation" methods first, and then some discussion of writing methods afterwords.